# Physical parameters
c : 1.484 # speed of sound in medium
receiver_locations: # position of non-reference receivers (reference (origin) not included)
  - [-25.4,     0, 25.4]
  - [ 25.4,     0,    0]
  - [    0, -25.4,    0]

# Input parameters
input_mode : serial
input_timeout : 1.0
port : "/dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_5_to_8_DMBP58-if01-port0"
baud : 115200
tx_request_code : 'B'
tx_start_code : "\xBB"
read_timeout : 1.0
scalar_size : 5
signal_size : 300
signal_bias : 32767.0
sampling_frequency : 486618
locating_frame : "map"
receiver_array_frame : "hydrophones"

# Algorithm parameters
method : "LS"
target_frequency : 35000
upsampling_factor : 20
min_variance : 200
observation_buffer_size : 200

